packages/components/src/draggable/index.js

```diff
@@ -15,14 +15,20 @@ const cloneWrapperClass = 'components-draggable__clone';
 const cloneHeightTransformationBreakpoint = 700;
 const clonePadding = 20;
 
+const isChromeUA = ( ) => /Chrome/.test( window.navigator.userAgent );
+const documentHasIframes = ( ) => [ ...document.body.querySelectorAll( 'iframe' ) ].length > 0;
```

Comment:
Shall we scope the iframe search to the `.editor-block-list__layout` element? I fear that if we do that it would break at some point if we decide to change the CSS classes we use.

Also, I'm not aware of any other areas that could potentially use iframes outside the editor. Is there any? If so, this would give false positives.

Reply:
We should scope the search at least to the `#editor` element. The WordPress.com notifications bar uses an iframe.